Sources/SwiftGRPC/Core/Channel.swift

```diff
@@ -132,18 +132,17 @@ public class Channel {
 }
 
 private extension Channel {
-  class ConnectivityObserver {
+  final class ConnectivityObserver {
```

Comment:
Given that the containing extension is `private` already, making the class `final` won't make a difference to the compiler and it's unlikely that anyone is going to subclass `ConnectivityObserver` in the same file. However, it won't hurt, either.

Reply:
Agreed that this won't make a difference to the compiler, but I do think it's nice to annotate classes with `final` by default as opposed to making them subclass-able by default, since it is clearer from the reader's perspective whether something is _currently_ being subclassed (since it could have another subclass elsewhere in the file in this particular example)

Reply:
That makes sense, thank you for explaining.